notarymysql/start

```diff
@@ -1,12 +1,40 @@
 #!/bin/bash
 set -e
 
-DB_NAME='notary'
+# This database is used by both of Notary-Server and Notary-Signer
+# the early days which we would not use it any longer.
+DB_NAME_OLD='notary'
+
+# Message which will be displayed when the database 'notary' exsit.
+DB_WARNING="
+=============== WARNING =================
+# The schema has changed.               #
+# People should migrate those tables in #
```

Comment:
"Make sure you migrate the tables in 'notary' to 'notaryserver' and 'notarysigner'"